### PR TITLE
Adding unit testing for export_data renewal_survey_answers command

### PR DESCRIPTION
**Pull request by haudquaquam**
**Changes**

- Added `TestRenewalSurveyResponses` in `coldfront.core.utils.tests.test_export_data.py`
  - 3 tests: JSON, CSV, allowance type
  - Uses the following hardcoded elements:
    - Hardcoded renewal survey with sample answers for BRC version of `ProjectRenewalSurveyForm`
    - `Allowance Year 2024 - 2025`
    - CSV test uses question 6 from the renewal survey

**Testing**

- Within `app-shell`, run the following: `python3 manage.py test coldfront.core.utils.tests.test_export_data.TestRenewalSurveyResponses`

### CODE REVIEW

File: coldfront/core/utils/tests/test_export_data.py [hunk 1]
```diff
@@ -999,3 +1005,181 @@ def test_get_new_project_survey_responses_allowance_type(self):
...
+    def setUp(self):
+        super().setUp()
+
+        fixtures = []
+        filtered_fixtures = []
+
+        allocation_period = AllocationPeriod.objects.get(name__exact='Allowance Year 2024 - 2025')
+        
+        for index in range(5):
+            pi = User.objects.create(
```
Comment (matthew-li) [inline coldfront/core/utils/tests/test_export_data.py:1022]: Is this needed? Looks like the `pi` variable is getting replaced later.

File: coldfront/core/utils/tests/test_export_data.py [hunk 2]
```diff
@@ -999,3 +1005,181 @@ def test_get_new_project_survey_responses_allowance_type(self):
...
+                username=f'test_user{index}', first_name='Test', last_name='User',
+                is_superuser=True)
+            
+            renewal_survey = {
```
Comment (matthew-li) [inline coldfront/core/utils/tests/test_export_data.py:1026]: We tend to use single quotes across the codebase (though it's not entirely consistent).

Also, can you ensure that lines are (generally) <= 80 characters?

File: coldfront/core/utils/tests/test_export_data.py [hunk 3]
```diff
@@ -999,3 +1005,181 @@ def test_get_new_project_survey_responses_allowance_type(self):
...
+        survey : survey to modify
+        '''
+        multiple_choice_fields = {}
+        form = ProjectRenewalSurveyForm()
+        for k, v in form.fields.items():
+            # Only ChoiceField or MultipleChoiceField (in this specific survey form) have choices 
+            if (isinstance(v, forms.MultipleChoiceField)) or (isinstance(v, forms.ChoiceField)):
+                multiple_choice_fields[k] = {_k: _v for _k, _v in form.fields[k].choices}
+        for question, answer in survey.items():
+                if question in multiple_choice_fields.keys():
```
Comment (matthew-li) [inline coldfront/core/utils/tests/test_export_data.py:1174]: Too much indentation.

**Review by matthew-li — CHANGES_REQUESTED** [review]
Thanks, these are passing for me and overall look fine. I left a few comments about code style.

**Comment by haudquaquam** [comment]
Thank you @matthew-li, I addressed your comments.

**Review by matthew-li — APPROVED** [review]

**Pull request merged**